nx-dev/ui-common/src/lib/version-picker.tsx

```diff
@@ -41,11 +41,11 @@ export function VersionPicker(): JSX.Element {
               <div className="relative">
                 <ListboxButton
                   className={
-                    'relative w-full cursor-pointer rounded-lg border border-slate-200 py-2 pl-3 pr-10 text-left font-medium focus:outline-none focus-visible:border-blue-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-blue-300 sm:text-sm dark:border-slate-700'
+                    'relative w-full cursor-pointer rounded-lg border-slate-200 py-2 pr-6 text-left font-medium focus:outline-none focus-visible:border-blue-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-blue-300 sm:text-sm dark:border-slate-700'
                   }
                 >
                   <span className="block truncate">{selected.label}</span>
-                  <span className="pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2">
+                  <span className="pointer-events-none absolute inset-y-0 right-0 flex items-center pr-1">
```

Comment:
```suggestion
                  <span className="pointer-events-none absolute inset-y-0 right-0 flex items-center">
```
You can remove the padding right here.